src/main/kotlin/dev/rosewood/roseskyblock/manager/IslandManager.kt

```diff
@@ -35,12 +34,17 @@ class IslandManager(rosePlugin: RosePlugin) : Manager(rosePlugin) {
         copy.removeIf { it <= 0 }
 
         var current = 1
-        for (i in copy) {
-            if (i == current) {
-                current++
-            } else break
+        copy.forEach {
+            if (it == current) current++
```

Comment:
I feel like this changes the output since it no longer has a break statement? If not, it probably just wastes a few extra loop cycles